### PR TITLE
chore: update CoC to match the current CNCF's CoC

### PR DESCRIPTION
**Pull request by yizha1**
Updates:
- Update CoC doc based on the [template](https://github.com/cncf/project-template/blob/main/CODE_OF_CONDUCT.md) 

Signed-off-by: Yi Zha <yizha1@microsoft.com>

### CODE REVIEW

**Review by priteshbandi — DISMISSED** [review]
LGTM

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,43 +1 @@
...
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting a CNCF project maintainer, Sarah Novotny <sarahnovotny@google.com>, and/or Dan Kohn <dan@linuxfoundation.org>.
-
-This Code of Conduct is adapted from the Contributor Covenant
-(https://contributor-covenant.org), version 1.2.0, available at
-https://contributor-covenant.org/version/1/2/0/
-
-### CNCF Events Code of Conduct
-
-CNCF events are governed by the Linux Foundation [Code of Conduct](https://events.linuxfoundation.org/events/cloudnativecon/attend/code-of-conduct) available on the event page. This is designed to be compatible with the above policy and also includes more details on responding to incidents.
+The Notary Project follows [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
```
Comment (TheFoxAtWork) [inline CODE_OF_CONDUCT.md:1]:
```suggestion
# The Notary Project Code of Conduct
The Notary Project follows [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
```

Comment (yizha1) [reply CODE_OF_CONDUCT.md:3]: Thanks @TheFoxAtWork I have updated the PR based your comment.

**Review by TheFoxAtWork — APPROVED** [review]
LGTM thank you for resolving this!

**Review by priteshbandi — APPROVED** [review]
LGTM

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Pull request merged**